metadata_backend/api/operators.py

```diff
@@ -625,8 +625,8 @@ async def update_folder(self, folder_id: str, patch: JsonPatch) -> str:
         await self._check_folder_exists(self.db_service, folder_id)
         try:
             folder = await self.db_service.read("folder", folder_id)
-            JSONValidator(folder, "folders").validate
             upd_content = patch.apply(folder)
+            JSONValidator(upd_content, "folders").validate
```

Comment:
This was meant to just be a sanity check so that the `folder` returned from `db_service.read()` is still fine. However, validating the patched content is probably pretty good to have. 👍

Reply:
Well, I don't think sanity check is needed after first read where content hasn't been changed yet. But doing it after patching content in db is a good one, I'll add that